iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/VoltageAngleLimitImpl.java

```diff
@@ -18,7 +18,7 @@
  * @author José Antonio Marqués {@literal <marquesja at aia.es>}
  * @author Bertrand Rix {@literal <bertrand.rix at artelys.com>}
  */
-class VoltageAngleLimitImpl implements VoltageAngleLimit {
+class VoltageAngleLimitImpl extends AbstractPropertiesHolder implements VoltageAngleLimit {
```

Comment:
Properties can be added on `VoltageAngleLimit`s, but they are not (de)serialized. Their support is missing in:
- `VoltageAngleLimitSerDe`
- the IIDM XSDs